app/code/Magento/InventoryApi/etc/webapi.xml

```diff
@@ -100,6 +100,18 @@
             <resource ref="Magento_InventoryApi::source_stock_link"/>
         </resources>
     </route>
+    <route url="/V1/inventory/stock/source-link/save" method="POST">
```

Comment:
Replace on `/V1/inventory/stock-source-link`

`save` in url is contradicts to REST concept
We have HTTP method for that